packages/core/src/util/slotCalc.ts

```diff
@@ -43,7 +45,12 @@ export const eraSummariesConfig: EraSummariesMap = {
 };
 
 const createSlotEpochCalcImpl = (eraSummaries: EraSummary[]) => {
-  const eraSummariesAsc = orderBy(eraSummaries, ({ start }) => start.slot);
+  // It's possible to configure when particular eras are upgraded, without an upgrade proposal, in
+  // testnet cardano-node configuration, including the specification of multiple eras in the same
+  // epoch. Era summaries therefore need to be filtered to remove eras that are being skipped,
+  // which is evidenced by a later era starting in the same slot.
+  const eraSummariesWithoutSkippedEras = Object.values(groupBy(eraSummaries, 'start.slot')).map(last) as EraSummary[];
```

Comment:
Is there any chance the eras in eraSummaries are not ordered by start.slot?
If there is, groupBy does not order them, so slot 5 will be before slot 0 if the era was start.slot 5 was before the one with start.slot 0